askama/src/lib.rs

```diff
@@ -55,20 +55,53 @@
 //!   specified extension (`ext`): if the extension is `html`, `htm` or `xml`,
 //!   the `html` escape mode is used; otherwise, no implicit escaping is done.
 //!   Setting an escape mode explicitly overrides the inferred value.
+//! * `syntax` (as `syntax = "foo"`): set the syntax name for a parser defined
+//!   in the configuration file. The default syntax , "default",  is the one
+//!   provided by Jinja.
 //!
 //! ## Configuration
 //!
 //! At compile time, Askama will read optional configuration values from
 //! `askama.toml` in the crate root (the directory where `Cargo.toml` can
-//! be found). Currently, this only covers the directories to search for
-//! templates.
+//! be found). Currently, this covers the directories to search for templates,
+//! as well as the personalization of syntax.
```

Comment:
Can we replace "personalization" with "customization" here? And please get rid of the extra empty line.